showkase-processor/src/main/java/com/airbnb/android/showkase/processor/ShowkaseProcessor.kt

```diff
@@ -489,3 +507,8 @@ internal enum class ShowkaseGeneratedMetadataType {
     TYPOGRAPHY
 }
 
+internal enum class ScreenshotTestType {
+    SHOWKASE,
```

Comment:
This is the `shot` one right? Would it make sense to name this `SHOT_SHOWKASE` maybe? :)